gobblin-modules/gobblin-kafka-common/src/test/java/org/apache/gobblin/source/extractor/extract/kafka/KafkaProduceRateTrackerTest.java

```diff
@@ -58,6 +59,7 @@ public void setUp() {
     kafkaPartitions.add(new KafkaPartition.Builder().withTopicName("test-topic").withId(1).build());
     this.workUnitState = new WorkUnitState();
     this.workUnitState.setProp(KafkaSource.RECORD_LEVEL_SLA_MINUTES_KEY, 5L);
+    this.workUnitState.setProp(ConfigurationKeys.KAFKA_BROKERS, "testBroker");
```

Comment:
Why do we set this in the tests? Do these tests even test the eventSubmitters?

Reply:
They do not, but the producerate/consumRate tracker call the util method which assume we do have broker defined.